src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -40,10 +40,9 @@ def load_arguments(self, _):
                                      completer=get_resource_name_completion_list('Microsoft.Compute/virtualMachineScaleSets'),
                                      help="Scale set name. You can configure the default using `az configure --defaults vmss=<name>`",
                                      id_part='name')
-    if self.supported_api_version(min_api='2018-06-01', operation_group='disks') or self.supported_api_version(min_api='2018-06-01', operation_group='virtual_machines'):
-        disk_sku = CLIArgumentType(arg_type=get_enum_type(['Premium_LRS', 'Standard_LRS', 'StandardSSD_LRS', 'UltraSSD_LRS']))
-    elif self.supported_api_version(min_api='2018-04-01', operation_group='disks') or self.supported_api_version(min_api='2018-06-01', operation_group='virtual_machines'):
-        disk_sku = CLIArgumentType(arg_type=get_enum_type(['Premium_LRS', 'Standard_LRS', 'StandardSSD_LRS']))
+
+    if StorageAccountTypes:
+        disk_sku = CLIArgumentType(arg_type=get_enum_type(StorageAccountTypes))
     else:
```

Comment:
let us get rid of this `else`

Reply:
@yugangw-msft, i added this because .get_models(storageaccounttypes) fails for some  profiles.

See this [job](https://travis-ci.org/Azure/azure-cli/jobs/443465561#L1341) in this previous CI [build](https://travis-ci.org/Azure/azure-cli/builds/443465552). Also see the other failed job in the build.

Reply:
I will open an issue in our repo and reference the service end bug.